src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Comment:
you already make it in App file, can we pass activeTab as props?

Reply:
Hey, @ArturErmolenko, sorry, I can't pass activeTab as props as the test expects activeTab to be found in the Tab component :( 

It works as expected (w/activeTab as props) – but with failed checks (please check if possible). Thank you!